spec/models/validations/sales/setup_validations_spec.rb

```diff
@@ -58,14 +61,6 @@
     end
 
     context "with sales_in_crossover_period == true" do
-      around do |example|
```

Comment:
Do we need to force `with_sales_in_crossever_period` to return true for this?

Reply:
Also, it looks like some of the tests in this context have some weird things (not in this PR) going on with their dates (in particular actual used dates vs mentioned in context names), and might fail once the current crossover period does finish or when the collection year rolls over - any chance you can take a look at them?

Reply:
Yeah this needed with_sales_in_crossever_period stub, the tests were also a bit wrong. Setting the new_logs_end_date and edit_end_date to before today sill passes these tests